themes/vue/layout/partials/language_dropdown.ejs

```diff
@@ -1,7 +1,9 @@
 <li class="nav-dropdown-container language">
   <a class="nav-link">言語</a><span class="arrow"></span>
   <ul class="nav-dropdown">
+    <li><a href="https://vuejs.org/" class="nav-link" target="_blank">English</a></li>
     <li><a href="https://vuejs.org/" class="nav-link" target="_blank">英語</a></li>
```

Comment:
Can you remove this line please? 
this line is duplicate with the below line. 😉